src/main/java/net/sf/mzmine/modules/rawdatamethods/peakpicking/manual/ManualPeak.java

```diff
@@ -309,4 +309,16 @@ public SimplePeakInformation getPeakInformation(){
     }
     //End dulab Edit
 
+	@Override
```

Comment:
Ya getting rid of that right now. It was the reason for the syntax errors that caused errors in the checks.